matrix-sdk-android/src/androidTest/java/org/matrix/android/sdk/api/TestMatrix.kt

```diff
@@ -35,16 +35,16 @@ import org.matrix.android.sdk.internal.SessionManager
 import org.matrix.android.sdk.internal.network.ApiInterceptor
 import org.matrix.android.sdk.internal.network.UserAgentHolder
 import org.matrix.android.sdk.internal.util.BackgroundDetectionObserver
+import org.matrix.android.sdk.internal.worker.MatrixWorkerFactory
 import org.matrix.olm.OlmManager
 import java.util.concurrent.Executors
 import java.util.concurrent.atomic.AtomicBoolean
 import javax.inject.Inject
 
 /**
- * This is the main entry point to the matrix sdk.
- * To get the singleton instance, use getInstance static method.
+ * This mimics the Matrix class but using TestMatrixComponent internally instead of regular MatrixComponent.
  */
-class Matrix private constructor(context: Context, matrixConfiguration: MatrixConfiguration) {
+class TestMatrix constructor(context: Context, matrixConfiguration: MatrixConfiguration) {
```

Comment:
Shouldn't it be `internal`?

Reply:
it's in androidTest package so not needed I guess